fidesctl/src/fidesctl/cli/commands/util_comands.py

```diff
@@ -0,0 +1,133 @@
+"""Contains all of the Utility-type CLI commands for Fidesctl."""
+import os
+
+import click
+import requests
+import toml
+
+from fidesctl.cli.options import yes_flag
+from fidesctl.cli.utils import (
+    handle_cli_response,
+    pretty_echo,
+)
+from fidesctl.core import api as _api
+from fidesctl.core.utils import echo_green, echo_red
+
+
+@click.command()
+@click.pass_context
+def init(ctx: click.Context) -> None:
+    """
+    Initialize a Fidesctl instance.
+    """
+
+    # Constants
+    dir_name = ".fides"
+    config_path = f"{dir_name}/fidesctl.toml"
```

Comment:
It may be helpful to split out the config file's name separately, and make these constants available at a higher scope for use elsewhere.
```suggestion
    config_dir = ".fides"
    config_file = "fidesctl.toml"
    config_file_path = f"{dir_name}/{config_file}"
```

Reply:
updated, but going to leave in this scope for now until we figure out a better place to put it